base_classes/NXwaveplate.nxdl.xml

```diff
@@ -58,13 +59,13 @@
     <!--Are there any other common wave plate types?-->
     <field name="other_type">
         <doc>
-             If you selected 'other' in type describe what it is.
+            If you selected 'other' in type describe what it is.
         </doc>
     </field>
     <field name="retardance">
         <doc>
-             Specify the retardance of the waveplate (e.g. full-wave, half-wave
-             (lambda/2), quarter-wave (lambda/4) plate).
+            Specify the retardance of the waveplate (e.g. full-wave, half-wave
+            (lambda/2), quarter-wave (lambda/4) plate).
         </doc>
         <enumeration>
             <item value="full-wave plate"/>
```

Comment:
Probably makes sense to omit "plate" in these choices

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.